LayoutTests/accessibility/aria-labelledby-hidden-text-expected.txt

```diff
@@ -0,0 +1,10 @@
+Tests that labels specified via aria-lablledby and aria-label are retrieved according to ARIA specifications in cases where the label text is hidden.
+
+PASS: axButton.title === `AXTitle: ${button.getAttribute('data-expectedlabel')}`
```

Comment:
should this appear in the expectation?

${button.getAttribute('data-expectedlabel')}

feels like we should get the interpreted results

Reply:
Our expect(expression, expectedValue) outputs the expectedValue without evaluating it. It does evaluate it to check equality with expression, but not when it outputs PASS or FAIL. That's how we wrote it originally. I thought about fixing this to make the expectation files more legible, but now I would have to update hundreds of -expected.txt files. I may still do it on a boaring hour, but would like to discuss with the team and agree on an output format to make sure we get it right this time and we don't have to do the mass update again in the future.